lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -18,6 +18,7 @@ module LogStash; module Outputs; class OpenSearch; class HttpClient;
   AWS_DEFAULT_REGION = 'us-east-1'
   AWS_IAM_AUTH_TYPE = "aws_iam"
   AWS_SERVICE = 'es'
+  BASIC_AUTH_TYPE = 'basic_auth'
```

Comment:
Why not just basic?

Reply:
BASIC_AUTH_TYPE helps to understand more clearly that it is an auth_type of type basic

Reply:
Do you mean the constant value as "basic" instead of "basic_auth"?

Reply:
Yes. Similar to OpenSearch CLI

Reply:
updated